config/config.go

```diff
@@ -132,6 +132,9 @@ func GetConfigurationValue(config *Config, key string) (string, error) {
 		return config.Author, nil
 	} else if lowerKey == "cpp.lib.location" {
 		return config.CppLibraryLocation, nil
+	} else if strings.HasPrefix(key,"config.templates") {
+    lang := key[strings.LastIndex(key,".") + 1:]
```

Comment:
The indentation doesn't look right to me, can you fix it ?

Reply:
yeah you're right, i'll fix that